content/introduction/use-cases/data-and-model-quality.md

```diff
@@ -28,6 +28,20 @@ Dolt is the only database with [branch](../../concepts/dolt/git/branch.md) and [
 
 Lastly, [commits](../../concepts/dolt/git/commits.md), [logs](../../concepts/dolt/git/log.md), and [diffs](../../concepts/dolt/git/diff.md) can be used for model insights. Did Thursday's model perform better than Tuesday's but had the same model weights? Inspect the data diff to see what changed. Inspect the commit log to see where that new data came from.
 
+# Dolt replaces...
+
+## Unstructured files in cloud storage
```

Comment:
For this one i was thinking companies often need all of these, and maybe DoltHub captures the vertical value chain. UI for analysts, hashes for reproducibility, OLTP engine for small metadata, locally clonable database for OLAP queries. Companies would need a bunch of different service providers otherwise.